back-end/src/reservations/reservations.controller.js

```diff
@@ -217,6 +219,30 @@ async function create(req, res) {
   res.status(201).json({ data: create[0] });
 }
 
+async function edit(req, res) {
+  const {
+    data: {
+      first_name,
+      last_name,
+      mobile_number,
+      reservation_date,
+      reservation_time,
+      people,
+    } = {},
+  } = req.body;
+  const updatedReservation = {
+    ...res.locals.reservation,
+    first_name: first_name,
```

Comment:
When the key of the object matches the variable for the  value of the object, you can shorthand this by just naming the key. `first_name` instead of `first_name: first_name`